changelog.md

```diff
@@ -1,12 +1,13 @@
 # Changelog
 
-## v0.6.0 | t.b.d.
+## v0.6.0 | 2020-08-18
+* Plot and return images and timestamps for scans using physical coordinate system rather than fast and slow scanning axis. **Note that this is a potentially breaking change for scans with the fast axis in y direction!**
+* `Scan.pixels_per_line` and `Kymo.pixels_per_line` also return the number of pixels along the fast axis now. **Note that this is a potentially breaking change for scans/kymos with the fast axis in y direction!**
+* Verify starting timestamp when reconstructing `Kymo` or `Scan`. In those cases, scans cannot be reliably reconstructed and an error is thrown. For kymos, the first (partial) line is omitted and a warning is issued. **Note that this is a potentially breaking change for scans and kymos where the scan/kymo was initiated before the exported time window!**
```

Comment:
`potentially breaking` sounds scary, but mostly because it's vague. Instead of `potentially breaking` just say `breaking` but specify the exact case when it does break something and how to resolve it. `potentially breaking` leaves a lot of room for uncertainty and will definitely prevent people from updating to the new version to avoid breaking their workflow. Specifying the exact scenario when the break happens and pointing out that this is not a common one at all should put a lot of readers at ease and also be more helpful to people who are actually affected by the change.

Reply:
Good point. Removed potentially.